inc/custom-header.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Sample implementation of the Custom Header feature
+ * http://codex.wordpress.org/Custom_Headers
+ *
+ * You can add an optional custom header image to header.php like so ...
+
+ *
+ * @package twentynineteen
+ */
+
+/**
+ * Set up the WordPress core custom header feature.
+ *
+ * @uses twentynineteen_header_style()
+ */
+function twentynineteen_custom_header_setup() {
+	add_theme_support( 'custom-header', apply_filters( 'twentynineteen_custom_header_args', array(
```

Comment:
Add the following line to enable header video support. See #243:

`		'video'                  => true,`